gallery/src/components/NxTable/NxTableFilterExample.tsx

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, {useState} from 'react';
+
+import {
+  NxTable,
+  NxTableBody,
+  NxTableCell,
+  NxTableHead,
+  NxTableRow,
+  NxFilterInput
+} from '@sonatype/react-shared-components';
+import { toLower } from 'ramda';
+
+const initialState = [
+  {name: 'Anna', country: 'USA'},
+  {name: 'Lean', country: 'France'},
+  {name: 'Louis', country: 'France'},
+  {name: 'Zach', country: 'Colombia'},
+  {name: 'Jimmy', country: 'Germany'}
+];
+
+const NxTableFilterExample = () => {
+
+  const [rows, setRows] = useState(initialState);
+  const [nameFilter, setNameFilter] = useState('');
+  const [countryFilter, setCountryFilter] = useState('');
+
+  const dropdownDefaultLabel = 'All countries';
```

Comment:
I believe this and the logic around it can be removed now

Reply:
Done: https://github.com/sonatype/sonatype-react-shared-components/pull/165/commits/1bf7da83e9d4d2802784de81267b17251d3e6c31